cmd/compute-domain-kubelet-plugin/device_state.go

```diff
@@ -425,6 +425,12 @@ func (s *DeviceState) applyComputeDomainChannelConfig(ctx context.Context, confi
 		return nil, fmt.Errorf("applyComputeDomainChannelConfig: unexpected results %v", results)
 	}
 
+	// For now, we treat each request as a request for channel zero, even if
+	// AllocationModeAll.
+	if err := s.allocateImexChannel(0); err != nil {
```

Comment:
from review: rename to `AssertImexChannelNotAllocated()` -- call further below, closer to other asserts.